src/helpers.ts

```diff
@@ -53,3 +53,11 @@ export const joinFieldRecursively = (fieldOrObject: any): string => {
     .join(' ')
   return joinedFields
 }
+
+export function stringParam(str: string) {
+  return JSON.stringify(str)
```

Comment:
How come you used  `JSON.stringify` over something like ``` `"${str}"` ```? Wouldn't the latter ensure the right thing is always done, and increase clarity?

Reply:
`"${str}"` will increase clarity, but fail when `str` contains `"`. `JSON.stringify` will encode special characters like `"`. 
```
> desc = `a complex "string"`
"a complex "string""
> `"${desc}"`
""a complex "string"""
> JSON.stringify(desc)
""a complex \"string\"""

```

Reply:
Ah, that's a very good point. that's fine then